themes/qgrid/layout/partials/nav.ejs

```diff
@@ -24,20 +24,26 @@
                     pages.sort(function (a, b) {
                         return a.order - b.order;
                     });
+                    var stringOftags = [];
 
                     for (var j = 0; j < pages.length; j++) {
                         var p = pages[j];
+                        if (p.tags){
+                            for (var index = 0; index < p.tags.length; index++){
+                                if (index==0){ stringOftags[j]=''};
+                                stringOftags[j] +=p.tags[index]+',';
+                            }}
         %>
-                        <li class="<%- page.title === p.title ? 'active-topic' : '' %> ">
+                        <li data-tag="<%- stringOftags[j]%>" class="<%- page.title === p.title ? 'active-topic' : '' %> ">
```

Comment:
can we do here just like `<%- p.tags ? p.tags.join(',') : ''` ? instead of all above lines?